tools/license_header.py

```diff
@@ -77,6 +77,7 @@
 
                # Licensed under Caffe header
                'src/operator/nn/pool.h',
```

Comment:
nit: move this to line 88 to avoid confusion

Reply:
done